config/defaults/server.cfg

```diff
@@ -1,5 +1,7 @@
 {
     "Name": "CodeCollaborate",
     "Port": 81,
-    "ProjectPath" : "./data/ProjectFiles/"
+    "ProjectPath" : "./data/ProjectFiles/",
+    "LogLevel": "Debug",
```

Comment:
Hmmm, with the changes to the runtests script, I think this should be "Info" so that travis doesn't print each request inline